manticore/ethereum.py

```diff
@@ -2237,7 +2237,7 @@ def flagged(flag):
                 if tx.sort == 'CALL':
                     if metadata is not None:
                         function_id = tx.data[:4]  # hope there is enough data
-                        function_id = binascii.hexlify(state.solve_one(function_id))
+                        function_id = binascii.hexlify(state.solve_one(function_id)).decode()
```

Comment:
https://github.com/trailofbits/manticore/pull/1009/files#diff-6cf4568b5e8712514d226faa04e417fbR634 ?

Reply:
if we want to ref them as bytes should probably just store them as bytes rather than decoding every item on each access to signatures.

since its a hex string, should probably just be str no? this is where python static typing stuff would be handy